docs/content/docs/tutorials/apigatewayv2-reference-example.md

```diff
@@ -0,0 +1,280 @@
+---
+title: "Manage HTTP API using the ACK APIGatewayv2 Controller"
+description: "Create and Invoke an Amazon APIGateway HTTP API using ACK APIGatewayv2 controller deployed on Amazon Elastic Kubernetes Service (EKS)."
+lead: "Create and invoke an Amazon APIGateway HTTP API using Amazon Elastic Kubernetes Service (EKS)."
+draft: false
+menu:
+  docs:
+    parent: "tutorials"
+weight: 43
+toc: true
+---
+
+The ACK service controller for Amazon APIGatewayv2 lets you manage HTTP APIs and VPC Links directly from Kubernetes.
+This guide will show you how to create and invoke an HTTP API using a single Kubernetes resource manifest.
+
+In this tutorial we will invoke a single public endpoint(https://httpbin.org/get) by fronting it with an HTTP API. You
+can add multiple Routes and Integrations to the same API and invoke many endpoints using the single HTTP API endpoint.
+HTTP API helps in creating frontend and organizing microservices using multiple Routes and Integrations.
```

Comment:
I think this last line is redundant - basically restating what the previous line said. Maybe you could talk about "we add one stage, which is set to auto deploy. To that we add a `GET` route..." etc.?